### PR TITLE
Added createVelocity command

### PR DESCRIPTION
**Pull request by ksyy**

### CODE REVIEW

**Comment by proyan** [comment]
Thanks Alexis. Could you also please add a createAccelerationSignal for the sake of completion?

File: src/sot-dynamic-pinocchio.cpp [hunk 1]
```diff
@@ -215,6 +215,12 @@ DynamicPinocchio( const std::string & name)
 				"string (signal name)","string (joint name)");
     addCommand("createVelocity",
 	       makeCommandVoid2(*this,&DynamicPinocchio::cmd_createVelocitySignal,docstring));
+ 
+    docstring = docCommandVoid2("Create an acceleration (matrix homo) signal only for one joint.",
```
Comment (proyan) [inline src/sot-dynamic-pinocchio.cpp]: Acceleration and velocity are not homogeneous matrices. They are vectors. Could you please correct the docs as well?

Comment (ksyy) [reply src/sot-dynamic-pinocchio.cpp]: Yeah, sorry, I copied the info from the function `createAccelerationSignal` but did not check if it was right…
I corrected both.

**Comment by proyan** [comment]
Thanks for the commits

**Pull request merged**